src/components/TodoItem.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import styles from './TodoItem.module.css';
+
+class TodoItem extends React.Component {
+  state = {
+    editing: false,
+  }
+
+  handleEditing = () => {
+    this.setState({
+      editing: true,
+    });
+  }
+
+  handleUpdatedDone = (event) => {
+    if (event.key === 'Enter') {
+      this.setState({ editing: false });
+    }
+  }
+
+  componentWillUnmount() {
+    console.log('Cleaning up...');
```

Comment:
please remove all console.logs because according to [JavaScript Best Practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md), It’s ugly, it kills performance and it can make confidential data be visible to anyone using the browser tools to look at your website.